source/jormungandr/jormungandr/scenarios/utils.py

```diff
@@ -207,3 +207,78 @@ def build_pagination(request, resp):
     'arrival_time': ArrivalJourneySorter,
     'departure_time': DepartureJourneySorter
 }
+
+
+def get_or_default(request, val, default):
+    """
+    Flask add all API param in the request dict with None by default,
+    so here is a simple helper to get the default value is the key
+    is not in the dict or if the value is None
+    """
+    return request.get(val, default) or default
```

Comment:
You have to check the return directly, in this case 0 or '' would be considered as None